docs/proposals/probabilistic_proofs.md

```diff
@@ -0,0 +1,241 @@
+
+_TODO(olshansky): Collect & provide data showing that the majority of the block space is taken up by proofs._
+_TODO(olshansky): Collect & provide data showing that the majority of block verification._
+
+## Attack Modelling
+
+In order to select the values for the three parameters, the attacker's likelihood of adversarial reward & penalty must be modeled.
+
+### Approach
+
+An _attack by example_ approach is used determining the appropriate values for `ProofRequestProbability`, `ProofRequirementThreshold` and `ProofMissingPenalty`. This will demonstrate the optimal malicious behaviour an attacker should follow and tend to that case.
+
+### Definitions
+
+A Bernoulli probability distribution will be used whereby each `Claim` & `Proof` pair can be treated as an independent Bernoulli Trial. When the `Claim` exceeds `ProofRequirementThreshold`, the model is _"short-circuited"_ and is therefore outside the sample space for this definition.
+
+The definition for success is taken from the Network's point of view.
+
+- **Success**: Servicer submits a false claim and gets caught
+- **Failure** (the remainder of the sample space excluding Success):
+  - Servicer submits a true claim and is required prove it
+  - Servicer a true claim and have no requirement to prove it
+  - Servicer submits a false claim and gets away with it
+  - Servicer submits a true claim, but fails to prove it
+
+<!--
+The types of questions we can ask are:
+
+- What is the stopping condition?
+- How many success until a single failure?
+- How many failures until a single success?
+- How many failures until N success?
+- How many success until N failures?
+- How much scalability does the network need?
+- With what likelihood should an attacker get away?
+-->
+
+### Example
+
+Assume:
+
+- `ProofRequirementThreshold = 100 POKT`
+- `ProofRequestProbability = 0.25`
+
+If the `Claim` is greater than or equal to `100 POKT`, a proof is mandatory and the model is _"short-circuited"_. Therefore, the attacker can _freeload_ by submitting claims for `99.99 POKT` and hope they never get caught. If they do get caught (i.e. `Success`), the burn (i.e. `ProofMissingPenalty`) should exceed the total reward accumulated.
+
+Since each claim is independent, an attacker would never submit a `Claim` exceeding `ProofRequirementThreshold`, and therefore have a `ProofRequestProbability` likelihood of being required to submit a proof.
+
+### Model
+
+A [Geometric_distribution](https://en.wikipedia.org/wiki/Geometric_distribution) is used to identify the probability of `k` failures (sample space containing an attacker getting away) until a single success (an attacker is caught).
+
+$$ p = ProofRequestProbability $$
+
+$$ q = 1 - p $$
+
+$$ Pr(X=k) = (1-p)^{k-1}p $$
+
+$$ k = \frac{ln(\frac{Pr(X=k)}{p})}{ln(1-p)} + 1 $$
+
+![download](https://user-images.githubusercontent.com/1892194/220803154-90dcdd6b-8141-40d2-9cca-ed27a995fcfb.png)
```

Comment:
This graph violates the support of the PDF. While the formula that you derived is not wrong, the grpah shows a support over all possible values of P, which results in real-valued Ks (that are not in the support of K={1,2,3,4,...}).
I would suggest to keep this graph discrete, showing only values for K that are in the support of Pr().

Reply:
![imagen](https://user-images.githubusercontent.com/141699/220908141-953be02a-f826-4bf5-a622-ccde0f580e50.png)
Only the dots are valid, seeing negative Ks in the Y/axis make my eyes hurt haha

Reply:
Great idea!! Totally on the same page regarding my 👀 hurting :)

I've updated the code in the Appendix (it's commented out in markdown) for future readers

<img width="734" alt="Screenshot 2023-02-23 at 6 21 33 PM" src="https://user-images.githubusercontent.com/1892194/221076333-f6578bc2-0567-4e9d-ae7f-8a483a86cc2d.png">